.github/dependabot.yml

```diff
@@ -1,28 +1,20 @@
-# To get started with Dependabot version updates, you'll need to specify which
-# package ecosystems to update and where the package manifests are located.
-# Please see the documentation for all configuration options:
 # https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 
 version: 2
 updates:
+
+  # Maintain dependencies for GitHub Actions
   - package-ecosystem: "github-actions"
-    # Workflow files stored in the
-    # default location of `.github/workflows`
     directory: "/"
     schedule:
-      interval: "weekly"
-
-  - package-ecosystem: "npm"
-    directory: "/modules/runner-binaries-syncer/lambdas/runner-binaries-syncer"
-    schedule:
-      interval: "weekly"
+      interval: "daily"
+    commit-message:
+      prefix: ".github:"
```

Comment:
Should we also add another section for the npm installs in 

- `/modules/runner-binaries-syncer/lambdas/runner-binaries-syncer` and 
- `modules/runners/lambdas/webhook` and
- `modules/webhook/lambdas/webhook`

here as they existed in the previous dependabot config?

Reply:
oh i didn't realize there was a file here before... i copied blindly. thanks for pointing this out.